rs/sns/swap/canister/canister.rs

```diff
@@ -454,6 +454,9 @@ fn canister_post_upgrade() {
     });
 
     init_timers();
+
+    // TODO[NNS1-3386]: Remove this line.
```

Comment:
Probably should not remove it until all existing swaps are migrated to have these fields populated?  Maybe put that in the comment.  

Do we know how many swaps have to be upgraded before we can remove this migration?

Reply:
> Do we know how many swaps have to be upgraded before we can remove this migration?

I haven't checked, but I could do that tomorrow if we need that info. Note that this migration isn't harmful as it's not doing anything for Swaps that do have the new fields set.